person/src/com/company/core/entrypoint/PersonFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.company.core.entrypoint;
+
+import com.company.core.model.Person;
+import com.company.core.service.impl.DancerActionImpl;
+import com.company.core.service.impl.ProgrammerActionImpl;
+import com.company.core.service.impl.SingerActionImpl;
+import com.company.core.service.interfacer.PersonAction;
+
+public class PersonFactory {
```

Comment:
Why it is is in entry point package ? 
There should not be any entry point in core . 
Factory should be in core but not entry point package

Reply:
done